benchmark/scala/src/main/scala/com/microsoft/tpch/ConvertTpchDatasetToParquet.scala

```diff
@@ -0,0 +1,139 @@
+/*
+ * Licensed to the .NET Foundation under one or more agreements.
+ * The .NET Foundation licenses this file to you under the MIT license.
+ * See the LICENSE file in the project root for more information.
+ */
+
+// The code is modified based on Wentao's code. Wentao's homepage: https://www.microsoft.com/en-us/research/people/wentwu/
```

Comment:
Do you have a link to the code? A bit weird to just link to someone's home page? @rapoth what's the process here to credit right people?

Reply:
Wentao's code in his internal branch, I asked him if him want to push the code and he replied that I can push the code and add his name in comment as credit : )
Yea, some question, @rapoth how to credit right people?